### ISSUE TITLE
Open the discussions tab

### ISSUE THREAD

**Issue by antoninhrlt** [tui-rs-revival/tui-rs-revival#52]
First of all, thank you for making this project revive.
I'm suggesting to enable the discussions on this repository. Indeed, people may want to ask questions, share news of this project, and perhaps announce that the original owner came back.
Well, it will be a great opportunity to exchange about this project.

**Comment by Nickiel12** [comment]
Hi @antoninhrlt ! Just so that you know, there is a discord server dedicated to this project. We are currently open to suggestions on future communication (like GitHub discussions), and I would recommend you bring them up!

https://discord.gg/SctraFAZ

**Comment by antoninhrlt** [comment]
Just joined the Discord.

**Comment by Xithrius** [comment]
I think it would still be a good idea to open up the discussion tab even if there's a Discord. Could you make it available anyway?

**Comment by mindoodoo** [comment]
Yeah the idea behind the discord would be that it needs to be **optional** for people to contribute.

I just enabled the discussions tab so I'll close this issue.
